config/routes/api/v2.rb

```diff
@@ -187,6 +187,7 @@
           end
         end
         get :freeip, :on => :member
+        get :import_ipv4_subnets, :on => :member
```

Comment:
This has side effects while GET requests should not have any. I'd expect POST or PUT.